mrp_production_project_estimated_cost/models/mrp_production.py

```diff
@@ -15,19 +15,41 @@
 #    along with this program.  If not, see http://www.gnu.org/licenses/.
 #
 ##############################################################################
-from openerp import models, api, exceptions, _
+from openerp import models, fields, api, exceptions, _
 
 
 class MrpProduction(models.Model):
     _inherit = 'mrp.production'
 
-    @api.one
+    active = fields.Boolean('Active', default=False)
+    name = fields.Char(
+        string='Referencen', required=True, readonly=True, copy="False",
+        states={'draft': [('readonly', False)]}, default="/")
+
     @api.model
-    def action_confirm(self):
+    def create(self, values):
+        sequence_obj = self.pool['ir.sequence']
+        if values['active']:
+            values['name'] = sequence_obj.get(self._cr, self._uid,
+                                              'mrp.production')
+        else:
+            values['name'] = sequence_obj.get(self._cr, self._uid,
+                                              'fictitious.mrp.production')
+        production = super(MrpProduction, self).create(values)
+        return production
+
+    @api.multi
+    def action_compute(self, properties=None):
+        self.ensure_one()
+        self._calculate_production_estimated_cost()
```

Comment:
Tienes toda la razón, se me había pasado llamar al método super, pero no tengo que discernir entre si es unf OF normal, o simulada, ya que esta función solo cargar las líneas de las operaciones a realizar.